package.json

```diff
@@ -1,82 +1,82 @@
-        "grunt-contrib-compress": "^1.4.1",
-        "grunt-contrib-copy": "0.4.1",
-        "grunt-contrib-htmlmin": "0.1.3",
-        "grunt-contrib-jshint": "0.6.0",
-        "grunt-exec": "0.4.6",
-        "grunt-postcss": "0.3.0",
-        "grunt-contrib-less": "1.4.0",
-        "grunt-contrib-requirejs": "0.4.1",
-        "grunt-contrib-uglify": "2.1.0",
-        "grunt-git": "^0.3.4",
-        "grunt-npm": "git://github.com/sedge/grunt-npm.git#branchcheck",
-        "grunt-shell": "^1.1.2",
-        "grunt-targethtml": "0.2.6",
-        "grunt-text-replace": "^0.4.0",
-        "grunt-update-submodules": "^0.4.1",
-        "grunt-usemin": "0.1.11",
-        "habitat": "^3.1.2",
-        "load-grunt-tasks": "3.5.0",
-        "q": "1.4.1",
-        "semver": "^4.1.0",
-        "xmldoc": "^0.1.2",
-        "grunt-cleanempty": "1.0.3"
-    },
-    "devDependencies": {
-        "bluebird": "3.2.2",
-        "gunzip-maybe": "^1.3.1",
-        "http-server": "^0.9.0",
-        "mkdirp": "0.5.1",
-        "nunjucks": "2.3.0",
-        "properties-parser": "0.3.1",
-        "request": "^2.69.0",
-        "requirejs": "2.1.22",
-        "sw-precache": "^3.1.1",
-        "tar-fs": "^1.11.1"
-    },
-    "scripts": {
-        "localize": "node scripts/properties2js",
-        "postinstall": "grunt install",
-        "build": "grunt build-browser",
-        "preproduction": "grunt build-browser-compressed && grunt unlocalize",
-        "production": "npm start",
-        "unlocalize": "grunt unlocalize",
-        "test": "grunt test && grunt build-browser-compressed",
-        "start": "http-server -p 8000 --cors"
-    },
-    "license": "MIT"
+  "name": "bramble",
+  "version": "1.9.0-0",
+  "apiVersion": "1.9.0",
+  "homepage": "https://mozillathimblelivepreview.net/bramble/staging/dist/hosted.html",
+  "issues": {
+    "url": "http://github.com/mozilla/brackets/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mozilla/brackets.git",
+    "branch": "",
+    "SHA": ""
+  },
+  "dependencies": {
+    "anymatch": "1.3.0",
+    "autoprefixer": "^6.0.0",
+    "aws-sdk": "^2.19.0",
+    "chokidar": "1.6.0",
+    "glob": "7.0.6",
+    "grunt": "0.4.5",
+    "grunt-cleanempty": "1.0.3",
+    "grunt-cli": "0.1.9",
+    "grunt-contrib-clean": "0.4.1",
+    "grunt-contrib-compress": "^1.4.1",
+    "grunt-contrib-concat": "0.3.0",
+    "grunt-contrib-copy": "0.4.1",
+    "grunt-contrib-cssmin": "0.6.0",
+    "grunt-contrib-htmlmin": "0.1.3",
+    "grunt-contrib-jasmine": "0.4.2",
+    "grunt-contrib-jshint": "^1.1.0",
+    "grunt-contrib-less": "1.4.0",
+    "grunt-contrib-requirejs": "0.4.1",
+    "grunt-contrib-uglify": "2.1.0",
+    "grunt-contrib-watch": "0.4.3",
+    "grunt-eslint": "18.1.0",
+    "grunt-exec": "^1.0.1",
+    "grunt-git": "^0.3.4",
+    "grunt-jasmine-node": "0.1.0",
+    "grunt-lib-phantomjs": "0.3.0",
+    "grunt-npm": "git://github.com/sedge/grunt-npm.git#branchcheck",
```

Comment:
Yeah I'm pretty sure. I actually think we can even remove it entirely because the grunt task that uses it, we don't really use anymore.